src/qcodes_contrib_drivers/drivers/Kiutra/L_Type_Rapid.py

```
@@ -0,0 +1,430 @@
+
+        if not isinstance(value, (int, float, np.integer, np.floating)):
+            raise TypeError(f"{value!r} is not an int or float; {context}")
+        setpoint = self._temperature_setpoint_getter()
+        for temp_min, temp_max, ramp_max in self._ramp_limits:
+            if temp_min <= setpoint < temp_max:
+                self._valid_values = (0, ramp_max)
+                if not (0 <= value <= ramp_max):
+                    raise ValueError(
+                        f"Ramp rate {value} K/min is out of bounds for setpoint {setpoint} K. "
+                        f"Valid range is 0 to {ramp_max} K/min.; {context}"
+                    )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)